xml/ha_storage_protection.xml

```diff
@@ -332,6 +332,15 @@
         is considered delivered. The timeout should be long enough for the node to
         detect that it needs to self-fence.
        </para>
+       <para>
+        However, if the <literal>msgwait</literal> timeout is too long,
```

Comment:
I'd expect they had a good reason to configure it to be this "long" :-) How about to put it like "relatively long"?

Reply:
Yes, that's a bit vague. I could use the word "relatively", but then it raises the question in regards to what. ;-) For the time being, I use "relatively". :)

Reply:
I'd say basically in regard of how fast the nodes boot/rejoin ;)